boxsdk/client.py

```diff
@@ -105,7 +105,7 @@ def users(self):
         response = box_response.json()
         return [User(self._session, item['id'], item) for item in response['entries']]
 
-    def search(self, query, limit, offset, ancestor_folders=None, file_extensions=None):
+    def search(self, query, limit, offset, ancestor_folders=None, file_extensions=None, mdfilters=None):
```

Comment:
I know it's called mdfilters in the API, but can you call the parameter metadata_filters?